linux_os/guide/services/snmp/snmp_configure_server/snmpd_no_rwusers/bash/shared.sh

```diff
@@ -0,0 +1,5 @@
+# platform = multi_platform_wrlinux,Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8
```

Comment:
You proposed to enable the rule also on Debian 9, Debian 10, RHEL 6 and Fedora, but the Bash script is more restricted. Is the SNMP configuration different on these distributions or is this an omission?

Reply:
For the moment I'll restrict this to RH like systems as Debian can put things into a number of locations.  My initial plan was to be more aware, but during devel I hit a snag with Debian... forgot to drop them...